src/test/EmailTest/EmailUtilTest.java

```diff
@@ -0,0 +1,57 @@
+package EmailTest;
+
+import Security.EmailExceptions;
+import Security.EmailUtil;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class EmailUtilTest {
+
+  private EmailUtil emailUtil = new EmailUtil();
+
+  @Test
+  public void checkOrgInvite() {
+    try {
+      String actual = emailUtil.getOrganizationInviteEmail("ji", "Cat", "Kap");
+      // System.out.println(actual);
+    } catch (EmailExceptions e) {
+      System.out.print(e.toString());
+    }
+  }
+
+  @Test
+  public void checkVeriCode() {
+    try {
+      String actual = emailUtil.getVerificationCodeEmail("hiii");
+      // System.out.println(actual);
+    } catch (EmailExceptions e) {
+      System.out.print(e.toString());
```

Comment:
@cccyyyr hii i would actually instead of printing the exception i would do something with assertThrows https://www.baeldung.com/junit-assert-exception to check the exception instead of printing it out